Cargo.lock

```diff
@@ -6168,6 +6168,7 @@ dependencies = [
  "bincode",
  "chrono",
  "color-eyre",
+ "crossbeam-channel",
```

Comment:
The format upgrade runs ~10x faster for me now, about 8 minutes for 2M blocks, which has been helpful for manual testing.